var/spack/repos/builtin/packages/comgr/package.py

```diff
@@ -16,6 +17,7 @@ class Comgr(CMakePackage):
     url      = "https://github.com/RadeonOpenCompute/ROCm-CompilerSupport/archive/rocm-5.1.3.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala', 'haampie']
+    libraries = ['libamd_comgr.so']
```

Comment:
@alalazo do we have something generic here? `.so/.dylib/.dll`?

Reply:
@haampie @alalazo @srekolam I think that the library names should not contain a suffix. During the search, spack should add the appropriate suffix based on the underlying OS.

Reply:
@haampie @alalazo @srekolam Okay, so I have verified that we can just put in the library without the suffix.  Spack will add it during the search.  However, the `determine_version` function is specific to finding based on the suffix available, which I think is fine.